pynucastro/networks/rate_collection.py

```diff
@@ -568,6 +568,8 @@ def _build_collection(self):
             elif isinstance(r.chapter, int):
                 if r not in self.reaclib_rates:
                     self.reaclib_rates.append(r)
+                    if r.get_rate_id() == "n --> p <wc12_reaclib_weak_>":
+                        warnings.warn("ReacLib neutron decay rate (<n_to_p_weak_wc12>) does not account for degeneracy at high densities. Consider using tabular rate from Langanke.")
```

Comment:
I'd recommend pulling the message out into a variable, so it doesn't show up twice (see https://pypi.org/project/flake8-errmsg/ for more reasoning)
```suggestion
                        msg = "ReacLib neutron decay rate (<n_to_p_weak_wc12>) does not account for degeneracy at high densities. Consider using tabular rate from Langanke."
                        warnings.warn(msg)
```